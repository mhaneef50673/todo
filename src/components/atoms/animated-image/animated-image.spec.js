import React from 'react';
import { mount } from 'enzyme';
import { mountToJson} from 'enzyme-to-json';
import AnimatedImage from './index';

describe('Animated Image Component', () => {
  it('should render the component', () => {
    const tree = mount(
      <AnimatedImage imageUrl="testUrl" extraClass="extraClass" showRippleEffect shouldUseRippleEffect />
    )
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});