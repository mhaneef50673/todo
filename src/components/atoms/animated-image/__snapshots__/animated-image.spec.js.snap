// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Animated Image Component should render the component 1`] = `
<AnimatedImage
  extraClass="extraClass"
  imageUrl="testUrl"
  shouldUseRippleEffect={true}
  showRippleEffect={true}
>
  <div
    className="image-container"
  >
    <img
      alt="animated"
      className="extraClass"
      src="testUrl"
    />
    <span
      className="ripple rippleEffect"
    />
  </div>
</AnimatedImage>
`;
